Library/Homebrew/formula_creator.rb

```diff
@@ -116,8 +118,8 @@ class #{Formulary.class_s(name)} < Formula
           # depends_on "cmake" => :build
         <% end %>
 
-        <% if mode == :python %>
-          # Additional Python dependency
+        <% if mode == :python or mode == :perl %>
```

Comment:
Switch `:perl` and `:python` here. Also, it seems like this ERB is the only place where we are using `or` instead of `||`.

Reply:
Yeh, would be nice to use `||` consistently throughout the entire fire if you don't mind @dawidd6!